post-processor/docker-import/post-processor.go

```diff
@@ -22,6 +22,7 @@ const BuilderId = "packer.post-processor.docker-import"
 type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
 
+	Executable string   `mapstructure:"docker_path"`
```

Comment:
One note is that users have to set the executable for every builder/post processor, it'd be cool if there was a concept of a plugin wide setting, sort of how Terraform allows provider settings to be defined in a provider block..  Not really something that the Packer SDK supports, it just would be nice for values like this that need to be shared across multiple Packer resources.

Reply:
Yeah I agree, we see that also with other plugins, this would indeed be a nice-to-have in the future, but we do need to think about how to define and scope them.